Meadowlark-js/tests/http/local.33b.http

```diff
@@ -110,7 +110,7 @@ content-type: application/json
 
 ###
 @contentClassDescriptorLocation = {{contentClassDescriptor.response.headers.location}}
-GET http://localhost:3000{{contentClassDescriptorLocation}}
+GET http://localhost:3000/{{contentClassDescriptorLocation}}
```

Comment:
The old code was correct - should not have a slash here, as the value in the location header already starts with a slash.
```suggestion
GET http://localhost:3000{{contentClassDescriptorLocation}}
```